PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerRunHandler.java

```diff
@@ -103,11 +108,16 @@ public Object onExecute(ExecutionEvent event) throws ExecutionException {
             // export WAR file
             ConsoleLogger.info(String.format(Constant.MESSAGE_EXPORTING_PROJECT, destinationPath));
             if (MavenUtils.isMavenProject(project)) {
-                MavenUtils.executePackageGoal(project);
+                MavenExecuteAction action = new MavenExecuteAction(MAVEN_GOALS);
+                IContainer container;
+                container = MavenUtils.getPomFile(project).getParent();
```

Comment:
container will always be got?

Reply:
in try/catch block, will print error and send telemetry